aspnet/client-side/yeoman.rst

```diff
@@ -1,95 +1,92 @@
-- `ASP.NET generator <https://www.npmjs.com/package/generator-aspnet>`_ creates ASP.NET 5 DNX projects.
-- You can even `generate your own Yeoman generator <https://github.com/yeoman/generator-generator>`_.
-
-In this article:
-  - `Getting Started`_
-  - `Building and Running from Visual Studio`_
-  - `Client-Side Build Support`_
-  - `Restoring, Building, and Hosting from the Command Line`_
-  - `Adding to Your Project with Sub Generators`_
-
-Getting Started
----------------
-The `ASP.NET generator <https://www.npmjs.com/package/generator-aspnet>`_ creates ASP.NET 5 DNX projects that can 
-be loaded into Visual Studio 2015 or run from the command line. The generator creates the following project types:
-
-- **Empty Application**: An empty Web application with minimal dependencies.     
-- **Console Application**: A DNX-based console application.
-- **Web Application**: A complete MVC web application with a simple home page and examples for managing accounts and login.
-- **Web Application Basic**: A simpler MVC web application lacking membership and authorization, among other advanced features which are found only in the **Web Application** generator.   
-- **Web API Application**: A Web API built with MVC.   
-- **Nancy ASP.NET Application**: A lightweight HTTP service with one module built using `Nancy <http://nancyfx.org/>`_.
-- **Class Library**: A DNX-based class library.
-- **Unit Test project**: A simple `XUnit <http://xunit.github.io/>`_ test project. 
-
-This walkthrough demonstrates how to use Yeoman to generate an ASP.NET 5 web application. 
+- Install `Node.js <https://nodejs.org/en/>`__. The installer includes `Node.js <https://nodejs.org/en/>`__ and `npm <https://www.npmjs.com/>`__.
  
-1. Follow the instructions on http://yeoman.io/learning/ to install **yo** and other required tools. 
+- Follow the instructions on http://yeoman.io/learning/ to install `yo <https://github.com/yeoman/yo>`__, bower, grunt, and gulp.
+
+	- ``npm install -g yo bower grunt-cli gulp``
+	
+.. note:: If you get the error ``npm ERR! Please try running this command again as root/Administrator.`` Run the command using `sudo <https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man8/sudo.8.html>`__: ``sudo npm install -g yo bower grunt-cli gulp``
 
-2. On the command line, install the ASP.NET generator:  
+- From the command line, install the ASP.NET generator: 
 
 .. code-block:: console
 
  npm install -g generator-aspnet
+ 
+.. note:: If you get a permission error, run the command under ``sudo`` as described above.
 
-.. note:: The ``–g`` flag installs the generator globally, so that it can be used from any path on your system.
+- 	- The ``–g`` flag installs the generator globally, so that it can be used from any path.
+
+Create an ASP.NET app
+-------------------------
  
-3. Create the new directory in which the project will be generated:
+- Create a directory for the project
 
 .. code-block:: console
 
  mkdir C:\MyYo
```

Comment:
Can you do a pass on this doc to change all the single space indentations to be two spaces?